### PR TITLE
Add multiline entry

### PR DESCRIPTION
**Pull request by convolu**
This adds the Multiline Entry widget to the toolset.

This widget was developed for Make Light. It is similar to a Textview, but has such features as:
* Kano styling
* Default placeholder text
* Maximum character limit

### CODE REVIEW

**Comment by convolu** [comment]
@tombettany @pazdera @radujipa Could you please review?

File: kano/gtk3/multiline_entry.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    @property
+    def max_length(self):
+        return self._max_length
+
+    @max_length.setter
+    def max_length(self, value):
+        if value <= 0:
+            self._max_length = 0
+        else:
+            self._max_length = value
```
Comment (tombettany) [inline kano/gtk3/multiline_entry.py]: Is this not:

```python
self._max_length = max(value, 0)
```

Comment (convolu) [inline kano/gtk3/multiline_entry.py]: Indeed it is

**Comment by tombettany** [comment]
Small comment but otherwise :+1:

**Comment by convolu** [comment]
Corrected as per your suggestion @tombettany, thanks

**Comment by pazdera** [comment]
:shipit:

**Comment by convolu** [comment]
Thank you for reviewing.

**Pull request merged**